packages/generators/app/src/resources/files/js/config/plugins.js

```diff
@@ -0,0 +1,2 @@
+module.exports = () => ({
```

Comment:
The default should pass in the env function like some of the other config templates.

Reply:
Hello @derrickmehaffy when I pass the env function without using it like that : 
```
export default ({ env }) => ({
});
```
 I then have unused variable error in CI, how should I solve this ?

Reply:
I'm guessing that's something wrong with the linting this file should be excluded from that.

Any thoughts @alexandrebodin?

Reply:
Hello @alexandrebodin @derrickmehaffy any input on how to move forward with this ?

Reply:
the lint error is fine and the initial proposal is good for now. We should using typescript typings to help discovering that env param later but not necessary here

Reply:
okay thanks I am modifying the PR to drop the last commits and go back to initial proposal